pkg/cri/server/helpers_linux_test.go

```diff
@@ -29,32 +29,38 @@ import (
 
 func TestGetCgroupsPath(t *testing.T) {
 	testID := "test-id"
-	for desc, test := range map[string]struct {
+	for _, test := range []struct {
+		desc          string
 		cgroupsParent string
 		expected      string
 	}{
-		"should support regular cgroup path": {
+		{
+			desc:          "should support regular cgroup path",
 			cgroupsParent: "/a/b",
 			expected:      "/a/b/test-id",
 		},
-		"should support systemd cgroup path": {
+		{
+			desc:          "should support systemd cgroup path",
 			cgroupsParent: "/a.slice/b.slice",
-			expected:      "b.slice:cri-containerd:test-id",
-		},
-		"should support tailing slash for regular cgroup path": {
+			expected:      "b.slice:cri-containerd:test-id"},
+		{
```

Comment:
Looks like `},` got omitted

Reply:
Thanks! I will update it in the follow-up.